flang/runtime/clock.h

```diff
@@ -26,8 +26,12 @@ extern "C" {
 /// - Windows implementation (currently does nothing)
 void RTNAME(DateAndTime)(char *date, char *time, char *zone,
     std::size_t dateChars, std::size_t timeChars, std::size_t zoneChars);
-}
 
-// TODO: CPU_TIME, SYSTEM_CLOCK
+/// Implement runtime for SYSTEM_CLOCK intrinsic.
+void RTNAME(SystemClock)(int *count, int *count_rate, int *count_max);
```

Comment:
I think it might be better to use `long long*` to avoid counter overflow as much as possible here. `count` may be negative in case the runtime cannot measure time, so at least this one needs to be signed.